lego/utils/serializers.py

```diff
@@ -14,11 +14,13 @@ class GenericRelationField(serializers.CharField):
     }
 
     def __init__(self, *args, **kwargs):
-        kwargs["write_only"] = True
         super().__init__(*args, **kwargs)
 
     def to_representation(self, value):
-        return None
+        try:
+            return instance_to_string(value)
+        except Exception:
+            pass
```

Comment:
Dont know why `GenericRelationField` was write-only, it prevented comments from returning `content_target`. If it should stay how it was, it is easy enough to add it to the websocket meta data instead of this.